Added clarification in the online documentation that log_* configuration is for supportability only (#939)
elastic/apm-agent-php: docs/configuration.asciidoc

elastic_apm.service_name=SERVICE_NAME
----

[float]
[[configure-apm-data-amount]]

> === Controlling amount of APM data emitted by the agent

Reviewer: In general, we try to avoid gerunds in headings (words that end in `ing`). What about this?

```suggestion
=== Control the amount of APM data emitted by the agent
```

> The following configuration settings control amount of APM data emitted by the agent
> and they can be tuned to reduce the agent overhead on the monitored application,
> network utilisation and amount of storage reuired by Elasticsearch.

Reviewer: ```suggestion
The following configuration settings control the amount of APM data emitted by the agent.
They can be tuned to reduce the agent overhead on the monitored application,
network utilization, and the amount of storage required by {es}.
```

- <<config-transaction-sample-rate>>
- <<config-breakdown-metrics>>

[float]
[[configure-logging]]
=== Logging

NOTE: Configuration settings described in this section are related to logs emitted by the agent itself
and not the logs emitted by the monitored application.
The intended use case for these configuration settings is for the agent's supportability.

> The logs emitted by the agent are not stored in Elasticsearch by default
> so reducing the level for these logs will not reduced the amount of storage used Elasticsearch -

Reviewer: ```suggestion
The logs emitted by the agent are not stored in Elasticsearch by default,
so reducing the level for these logs will not reduce the amount of storage used in Elasticsearch--
```
